### ISSUE TITLE
contains_only seems to be not working 

### ISSUE THREAD

**Issue by marooned071** [ActivisionGameScience/assertpy#77]
Hi,

I'm having trouble with `contains_only` method from assertpy. My understanding is that this assertion should check if two iterables has same elements: `is set(list1) equal to set(list2)`. In fact, this method is checking if iterable under tests is a subset of an argument of `contains_only`. `assertpy` version is `0.12` 

```py
import unittest
from assertpy import assert_that

class AssertPyTestContainsOnly(unittest.TestCase):
    def test_contains_only(self):
        some_list = ["a", "b", "c"]
        assert_that(some_list).contains_only("x", "a", "b", "c")
```

This test is passing but `some_list` doesn't contain `x`. Is should it behave like this? If yes, the name is quite misleading.

**Comment by saturnboy** [comment]
@marooned071 you are correct, i'll fix it

**Comment by saturnboy** [comment]
We expect:

```py
assert_that([1,2,3]).contains_only(1,2,3,4)
```

To raise:

```
Expected <[1, 2, 3]> to contain only <1, 2, 3, 4>, but did not contain <4>.
```

But duplicates are ok, so this passes:

```py
assert_that((1,1,1,1)).contains_only(1)
```

**Comment by saturnboy** [comment]
@marooned071 fixed.  this will be included in v0.13.  expect that release soon

**Comment by marooned071** [comment]
@saturnboy , thanks a lot for fixing! Thanks for the assertpy this is a really great and comfortable lib. We are using it across the whole project.  Good work guys! :)
